src/content/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring.mdx

```diff
@@ -123,37 +99,24 @@ Real-time metrics and analytics reduce your [mean-time-to-resolution](https://ne
+From the <DoNotTranslate>**Events**</DoNotTranslate> page you can track config changes, restarts, SSH sessions, and other [key event](/docs/infrastructure-events-page#types) changes. A real-time feed gives you a changelog for your entire infrastructure.
 
 <img
   title="New Relic - Infrastructure monitoring - Events"
   alt="New Relic - Infrastructure monitoring - Events"
   src={infrastructureEvents}
 />
-
 <figcaption>
   The <DoNotTranslate>**Events**</DoNotTranslate> page contains a real-time feed of all that is happening in your hosts.
 </figcaption>
 
-With our infrastructure monitoring tools you can also:
-
-* [Query your data](/docs/using-new-relic/data/understand-data/query-new-relic-data) to drill down into your [infrastructure events](/docs/default-infrastructure-events-attributes) and build [custom dashboards](/docs/dashboards/new-relic-one-dashboards/get-started/introduction-new-relic-one-dashboards) that you can share with your team.
-* Troubleshoot performance issues, no matter where the issue occurs—server-side or application-side—thanks to [APM data connection with infrastructure monitoring](/docs/infrastructure/new-relic-infrastructure/data-instrumentation/new-relic-apm-data-infrastructure).
-* [Create, view, or update alert settings](/docs/add-edit-or-view-host-alert-information) directly from the relevant infrastructure chart. For example, you can create a [host not reporting](/docs/infrastructure/new-relic-infrastructure/configuration/create-infrastructure-host-not-reporting-condition) alert condition.
-
-## View logs for your infrastructure and app data [#logs-context]
-
-Bring your logs and infrastructure data together to make troubleshooting easier and faster! With logs in context, you can see logs of your infrastructure data, such as Kubernetes clusters. You can add as many configuration files as you need; these files will serve as the sources that push your log metadata to our platform.
-
-You can also see log messages related to your errors and traces directly in your app's or host's UI. No need to switch to another UI page in New Relic! For more information, see our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context) documentation.
-
-If your <InlinePopover type="apm" /> agent doesn't support our automatic logs in context solution yet, you can continue to use our manual logs in context solutions, and [forward your logs via our infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) or other [supported third-party log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic).
+However you use it, our solution [securely](/docs/infrastructure/new-relic-infrastructure/getting-started/infrastructure-security) collects and displays your data in five seconds, so your monitoring never lags behind reality.
```

Comment:
Is it really five seconds? If it is that's great, otherwise it may be better to generalize this a bit more

Reply:
i think so but it's actually weird this is specified lol. im gonna remove it since i'm in the area (i didnt write this)